test_unstructured_ingest/evaluation-metrics.sh

```diff
@@ -14,31 +14,54 @@ EVAL_NAME="$1"
 # Download cct test from s3
 BUCKET_NAME=utic-dev-tech-fixtures
 FOLDER_NAME=small-eval-"$EVAL_NAME"
-LOCAL_EVAL_SOURCE_DIR=$SCRIPT_DIR/gold-standard/$FOLDER_NAME
-mkdir -p "$LOCAL_EVAL_SOURCE_DIR"
-aws s3 cp "s3://$BUCKET_NAME/$FOLDER_NAME" "$LOCAL_EVAL_SOURCE_DIR" --recursive --no-sign-request --region us-east-2
+SOURCE_DIR=$SCRIPT_DIR/gold-standard/$FOLDER_NAME
+mkdir -p "$SOURCE_DIR"
+aws s3 cp "s3://$BUCKET_NAME/$FOLDER_NAME" "$SOURCE_DIR" --recursive --no-sign-request --region us-east-2
 
 EXPORT_DIR="$SCRIPT_DIR"/metrics
 
 # shellcheck disable=SC1091
 source "$SCRIPT_DIR"/cleanup.sh
 function cleanup() {
   cleanup_dir "$OUTPUT_DIR"
-  cleanup_dir "$LOCAL_EVAL_SOURCE_DIR"
+  cleanup_dir "$SOURCE_DIR"
 }
 trap cleanup EXIT
 
+# build args
+function generate_args() {
+  local argtype="$1"
+  local dirpath="$2"
+  local list=("${@:3}")
```

Comment:
is the list required to exist to generate args? what happens in the case when no list is provided (will the loop / --${argtype}_list arg addition be skipped or empty)?

Reply:
hm, I see below that the output list argument is empty (but still provided). Just wanted to confirm the current/ideal behavior for if it's not supplied to generate args at all.

Reply:
The param --output_list will not be passing to the function then.